lib/DxilContainer/DxilContainerAssembler.cpp

```diff
@@ -1812,11 +1812,10 @@ class DxilRDATWriter : public DxilPartWriter {
           shaderKind = (uint32_t)props.shaderKind;
           if (pInfo2 && DM.HasDxilEntryProps(&function)) {
             const auto &entryProps = DM.GetDxilEntryProps(&function);
-            unsigned waveSize = entryProps.props.waveSize;
-            if (waveSize) {
-              pInfo2->MinimumExpectedWaveLaneCount = waveSize;
-              pInfo2->MaximumExpectedWaveLaneCount = waveSize;
-            }
+            pInfo2->MinimumExpectedWaveLaneCount = entryProps.props.waveMinSize;
+            pInfo2->MaximumExpectedWaveLaneCount =
+                entryProps.props.waveMaxSize > 0 ? entryProps.props.waveMaxSize
+                                                 : entryProps.props.waveMinSize;
```

Comment:
Ok, but missing use of `m_Module.GetWaveSize()` for actually filling in the PSV0 Min/Max values.  That's a good reason that method needed updating or removal.

Reply:
By this, I mean look around line 944 for where PSV0 is filled in.  That needs to be updated as well.